middleware/proxy/proxy.go

```diff
@@ -107,8 +109,16 @@ func (p Proxy) ServeHTTP(w http.ResponseWriter, r *http.Request) (int, error) {
 					}
 				}
 
+				var hiddenHeaders http.Header
+				if host.HiddenHeaders != nil {
+					hiddenHeaders = make(http.Header)
+					for header, _ := range host.HiddenHeaders {
+						hiddenHeaders.Add(header,"")
+					}
+				}
```

Comment:
Maybe it's obvious, but why do we have to iterate and copy each header here into another http.Header type?